include/internal/ktls.h

```diff
@@ -11,6 +11,17 @@
 # ifndef HEADER_INTERNAL_KTLS
 #  define HEADER_INTERNAL_KTLS
 
+#  ifndef EBADMSG
+#   define	EBADMSG		74	/* Not a data message */
```

Comment:
why is that necessary?

Reply:
Just in case these are not defined by glibc.

Reply:
but EMSGSIZE and EINVAL should always be defined.

Reply:
I don't think you can work around broken linux headers like this.
```
include/uapi/asm-generic/errno.h:#define	EBADMSG		74	/* Not a data message */
arch/alpha/include/uapi/asm/errno.h:#define	EBADMSG		84	/* Not a data message */
arch/mips/include/uapi/asm/errno.h:#define EBADMSG		77	/* Not a data message */
arch/parisc/include/uapi/asm/errno.h:#define	EBADMSG		67	/* Not a data message */
arch/sparc/include/uapi/asm/errno.h:#define	EBADMSG		76	/* Not a data message */
```